vars/main.yml

```
@@ -12,8 +12,25 @@ __mssql_server_ha_packages: mssql-server-ha
 __mssql_powershell_packages: powershell
 __mssql_conf_path: /var/opt/mssql/mssql.conf
 __mssql_conf_cli: /opt/mssql/bin/mssql-conf
+__mssql_ha_replica_types_all: [
```

```suggestion
__mssql_ha_replica_types_all:
  - primary
  - synchronous
  - asynchronous
  - witness
  - absent
```
Only use the `var: [elem1,elem2,...]` style for short, single line arrays - use the multiline yaml style otherwise - see also below

---

Thank you, I've applied this